src/style.css

```diff
@@ -0,0 +1,135 @@
+
+h2,
+p {
+  margin: 0;
+}
+
+body {
+  margin: 0;
+  --main-color: #060b35;
+  --secondary-color: #616070;
+  --blue-color: #00acdc;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+}
+
+.title {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: var(--main-color);
+}
+
+.card__title {
+  margin-top: 40px;
+}
+
+.product-code {
+  align-self: flex-start;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  color: var(--secondary-color);
+}
+
+.card__product-code {
+  margin-top: 4px;
+}
+
+.reviews {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+}
+
+.card__reviews {
+  margin-top: 16px;
+}
+
+.stars {
+  display: flex;
+  height: 16px;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  width: 16px;
+  margin: 0 2px;
+}
+
+.reviews__reviews-count {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  color: var(--main-color);
+}
+
+.stars--1 .stars__star:first-child,
+.stars--2 .stars__star:nth-child(-n+2),
+.stars--3 .stars__star:nth-child(-n+3),
+.stars--4 .stars__star:nth-child(-n+4),
+.stars--5 .stars__star:nth-child(-n+5) {
+  background-image: url(./images/star-active.svg);
+}
+
+.price {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.card__price {
+  margin-top: 24px;
+  margin-bottom: 16px;
+}
+
+.price__price-text {
+  font-size: 12px;
+  font-weight: 400;
+  color: var(--secondary-color);
+}
+
+.price__price-value {
+  font-size: 16px;
+  font-weight: 700;
+  color: var(--main-color);
+}
+
+.card__buy-button {
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 16px;
+  padding: 12px 56px;
+  text-transform: uppercase;
+  text-decoration: none;
+  color: #fff;
+  background-color: var(--blue-color);
+  border-radius: 5px;
+}
+
+.card__buy-button:hover {
+  color: var(--blue-color);
+  background-color: #fff;
+  border: 1px solid var(--blue-color);
```

Comment:
Add a border to the default state and on hover change border color only. In another case it's can make some trouble with size